### ISSUE TITLE
GridX: Intermittent problem loading metadata

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#2071]
@jhpedemonte @JonFerraiolo @peller 

This is a spin-off of the GridX work for #2034, but I'm wondering if cross-plugin issues related to #2067 are involved here...

I've noticed this off and on, but thought it was generally always tied to the occurrence of other JS errors. But, now that the GridX support is a little more solid and there are fewer extraneous exceptions, I'm convinced there's a problem here.

Pretty reliably (but not everytime) I can cause the error below, if I 1) restart the server and then quickly 2) clear the browser cache, 3) reopen Maqetta, 4) create an HTML file, and 5) drop GridX from the Untested widgets section of the Widget palette onto the new page. 

If I refresh the Maqetta page, then I can start using the GridX widget with no problem. So, t's almost as if I can "beat" the loading of all of the metadata when the server is first launched.

If you look closely at the error, it's saying it can't load ItemFileReadStore_oam.json (which lives in the Dojo 1.7 plugin) from the gridx plugin. I'm theorizing that eventually the metadata for Dojo 1.7 plugin gets read, and whatever caching is done gets the right copy of the ItemFileReadStore metadata in it (based on id of the widget).

```
Problem connecting to the Maqetta Server...
Return to Maqetta


Error description:

Unable to load app/metadata/gridx/1.0/dojo/data/ItemFileReadStore_oam.json?@revision@ status:404
```

**Comment by JonFerraiolo** [comment]
-> @jhpedemonte (so this doesn't fall through cracks)

**Comment by jhpedemonte** [comment]
Fixed by 5331530f8992f0d8fcdfb9107b5101b500de2690 on a branch.  I'll merge into master by Monday.
